tests/fatrace-btrfs

```diff
@@ -1,96 +0,0 @@
-#!/bin/sh
```

Comment:
test_btrfs(), the final unmount is more elaborate now with the retry loop. Dropped the `set -C`, it's a temp mount anyway. Checks are 1:1.